### DISCUSSION TITLE
Creative crafting, a new avenue for player creativity??

### DISCUSSION THREAD

**Discussion by DokimiCU** [jeremyshannon/Exile#121] [Ideas]
Basically I'm here spamming comments because I was checking out the dyes (nice feature! I finally get to feel what everyone else has the whole time: "What is this? I'm confused! Oh wait... it all makes sense. This is fun!")

Anyway, dyes and cooking pot reminded me of Oblivion where it was possible to create custom spells by combining various features. That kind of ability to make custom crafts opens up new possibilities of creativity.

e.g. for cooking, if the plant properties (Health effects) are transferred to the resulting brew then the player can make strange custom potions e.g. poisons, or healing energy drinks, or...? The craft becomes something unique that they made.

This could be built on with various herbalist or alchemical practices. e.g. the player has a herb garden and recipes they invented for brewing in their chemistry equipment. Different chemical treatments might add or remove properties. The player can go full mad scientist with their crazy brews.

Anyway, this is a very vague rambling thought (mostly I just wanted to say: well done with the new features!)

**Comment by jeremyshannon** [comment]
An alchemy-type system's not a bad idea!
